python/py-serial/Portfile

```diff
@@ -21,11 +21,11 @@ long_description    This module encapsulates the access for the serial port. \
                     The module named "serial" automatically selects the appropriate \
                     backend.
 
-checksums			rmd160  a307efb344e513f0f31efae1d0c3a1def3c68a9a \
-                    sha256  17279a02e2e14034f6366032b6e43db99ae6640b22b8fbac09cd47398a136173 \
-                    size 148412
+checksums           rmd160  ee56e22c15af484b72b65d445a4fe8a2aeee0cd0 \
+                    sha256  80125f950a620bd0a31cfa18f5eda0df77e0bed2accc2bb7a76ab650ff63afa9 \
+                    size    155864
 
-python.versions     27 35 36 37 38 39
+python.versions     27 34 35 36 37 38 39
```

Comment:
I am personally not in favor of (re-)adding EOL Python versions, but other than that looks good to me

Reply:
I don't think obsessing over EOLism makes any more sense for Python than it does for OS versions.  If EOLism were applied to macOS, MacPorts wouldn't run on anything older than 10.14.

In principle, MacPorts is a good platform for testing Python code across a wide range of Python versions, especially when the code might be used on other platforms with less flexibility in choosing Python versions.  But eradicating subports solely for being "too old" interferes with that.